homeassistant/components/mqtt/discovery.py

```diff
@@ -140,7 +141,33 @@ async def async_start(  # noqa: C901
 ) -> None:
     """Start MQTT Discovery."""
     mqtt_data = get_mqtt_data(hass)
-    mqtt_integrations = {}
+    platform_setup_lock: dict[str, asyncio.Lock] = {}
+
+    async def _async_jit_component_setup(
```

Comment:
let's not call this JIT, it suggests we're doing something fancy which we are not

Reply:
Addressed with: https://github.com/home-assistant/core/pull/113042